collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh

```diff
@@ -651,6 +651,10 @@ rke2-k8s() {
   if [[ ${RKE2_SERVER} && ! ${API_SERVER_OFFLINE} ]]; then
     KUBECONFIG=/etc/rancher/${DISTRO}/rke2.yaml
     ${RKE2_DATA_DIR}/bin/kubectl --kubeconfig=$KUBECONFIG api-resources > $TMPDIR/${DISTRO}/kubectl/api-resources 2>&1
+    if [ -f /etc/cni/net.d/05-cilium.conflist ]; then
```

Comment:
It looks like `05-cilium.conflist` is a consistently used filename, so we should be good to hard code this